CONTRIBUTING.md

```diff
@@ -159,6 +159,22 @@ Guidelines for a good tutorial:
 Note that the `Figure.show()` function needs to be called for a plot to be inserted into
 the documentation.
 
+### Example code standards
+
+When editing documentation, use the following standards to demonstrate the example code:
+
+1. Python arguments, such as import statements, Boolean expressions, and function 
+   arguments should be wrapped as ``code`` by using \`\` on both sides of the code.<br>
+   Example: \`\`import pygmt\`\` results in ``import pygmt``
+<br><br>
```

Comment:
Why do you need `<br>` here?

Reply:
When PyCharm would render the Markdown, there wouldn't be a line break without `<br>`.

Reply:
I don't think it's required by standard or github markdown.

Reply:
Removed the `<br>` tags